package.json

```diff
@@ -3443,7 +3443,7 @@
         "test": "npm run testCompile && ts-node ./scripts/test/test.ts && npm run report",
         "testE2E": "npm run testCompile && ts-node ./scripts/test/testE2E.ts && npm run report",
         "integrationTest": "npm run testCompile && ts-node ./scripts/test/integrationTest.ts && npm run report",
-        "lint": "eslint -c .eslintrc.js --ext .ts .",
+        "lint": "npm run testCompile && ts-node ./scripts/lint/testLint.ts",
```

Comment:
Is it possible to avoid `testCompile` here? That should normally be the responsibility of the caller. So `npm run lint` will be fast if I know that I already have an up-to-date `dist/`. It might be worth adding a sanity check in `testLint.ts` which checks that `dist/` exists and is non-empty.

Reply:
Updated to not run compile first, will look in to ensuring `dist` exists in a separate pr